src/components/download-button/__tests__/__snapshots__/download-button.test.js.snap

```diff
@@ -0,0 +1,85 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DownloadButton Default DownloadButton renders as expected 1`] = `
+<a
+  className="unprose btn btn--blue round-full py12 px24 txt-s my18"
+  download="airports.csv"
+  href="../files/airports.csv"
+>
+  <span
+    className="flex-parent flex-parent--center-cross"
+  >
+    <span
+      className="flex-child mr3"
+    >
+      <svg
+        className="events-none icon"
+        focusable="false"
+        role="presentation"
+        style={
+          Object {
+            "height": 18,
+            "width": 18,
+          }
+        }
+      >
+        <use
+          xlinkHref="#icon-arrow-down"
+          xmlnsXlink="http://www.w3.org/1999/xlink"
+        />
+      </svg>
+    </span>
+    <span
+      className="flex-child"
+    >
+      Download CSV
+    </span>
+  </span>
+</a>
+`;
+
+exports[`DownloadButton DownloadButton for PNG renders as expected 1`] = `null`;
+
+exports[`DownloadButton DownloadButton with optional text renders as expected 1`] = `null`;
```

Comment:
These are rendering as null, I don't think we expect that. This likely means that we need to check-in on how we're generating the snapshot in download-button.test.js.